### DISCUSSION TITLE
Google privacy page showing on weather app.

### DISCUSSION THREAD

**Discussion by spaceme0w** [valinet/ExplorerPatcher#859] [Q&A] [answered]
This is more likely an issue with the weather app, but I will report anyway just in case. This is how my weather app looks, it happens a few hours after a restart, it works ok initially. 

![Screenshot 2022-02-15 133810](https://user-images.githubusercontent.com/49492152/154055266-690facce-9685-48c9-a127-8ce0502d4697.jpg)

**Comment by Poopooracoocoo** [comment, marked as answer]
You may like to view 
https://github.com/valinet/ExplorerPatcher/discussions/838
